packages/lib/apps/getMostPopularApps.ts

```diff
@@ -0,0 +1,27 @@
+import { z } from "zod";
+
+import prisma from "@calcom/prisma";
+
+const getMostPopularApps = async () => {
```

Comment:
I think this function doesn't give most popular apps nor it does any ordered result.
It just gives the count of installs per appId. We should rename this accordingly.